assets/js/components/user-input/UserInputKeywords.js

```diff
@@ -120,8 +121,17 @@ export default function UserInputKeywords( { slug, max } ) {
 							'googlesitekit-user-input__text-option': values.length > i + 1 || value.length > 0,
 						} ) }
 					>
+						<VisuallyHiden>
+							<label
+								htmlFor={ `${ slug }-keyword-${ i }` }
+							>
+								{ __( 'Enter minimum one (1), maximum three (3) terms', 'google-site-kit' ) }
+							</label>
```

Comment:
@eugene-manuilov pointed out that this means each keyword input gets the same label, which isn't super-helpful. See: https://github.com/google/site-kit-wp/pull/2984#discussion_r601467779

Let's go with something like:

```js
<label htmlFor={ `${ slug }-keyword-${ i }` }>
	{ sprintf( __( 'Keyword %s', 'google-site-kit' ), i ) }
</label>
```